src/components/Card.js

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import "../scss/components/Card-mobile.scss";
+import "../scss/components/Card-tablet.scss";
+import "../scss/components/Card-desktop.scss";
+
+export default class Card extends React.Component {
+  state = {
+    products: this.props.data.products,
+  };
+
+  selectNormalCard = (index) => {
+    if (
+      this.state.products[index].isActive === false &&
+      this.state.products[index].isError === false
+    ) {
+      this.setState((state) => {
+        return (state.products[index].isActive = true);
+      });
+    } else {
+      this.setState((state) => {
+        return (state.products[index].isActive = false);
+      });
+    }
+  };
+
+  selectErrorCard = (index) => {
+    if (
+      this.state.products[index].isActive === false &&
+      this.state.products[index].isError === true
+    ) {
+      this.setState((state) => {
+        return (state.products[index].isActive = true);
+      });
+    } else {
+      this.setState((state) => {
+        return (state.products[index].isActive = false);
+      });
+    }
+  };
+
+  productCard = () =>
+    this.state.products.map((i, index) => (
```

Comment:
```suggestion
    this.state.products.map((product) => (
```
Zmienna `i` pod którą jest tak naprawdę obiekt produktu nie jest dobrą zmienną - `i` kojarzy się właśnie z indeksem. `index` nie potrzebujesz wcale, w komentarzu niżej piszę dlaczego 😄